From 499add1098733d5b45d46a56177e01792904bc80 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:23:57 +0100
Subject: [PATCH 01/39] journal: raise compression threshold.

---
 src/journal/journal-file.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -Naur systemd-239/src/journal/journal-file.c systemd-239.tpg/src/journal/journal-file.c
--- systemd-239/src/journal/journal-file.c	2018-06-22 11:11:49.000000000 +0000
+++ systemd-239.tpg/src/journal/journal-file.c	2018-07-12 14:13:37.423540227 +0000
@@ -33,7 +33,7 @@
 #define DEFAULT_DATA_HASH_TABLE_SIZE (2047ULL*sizeof(HashItem))
 #define DEFAULT_FIELD_HASH_TABLE_SIZE (333ULL*sizeof(HashItem))
 
-#define DEFAULT_COMPRESS_THRESHOLD (512ULL)
+#define DEFAULT_COMPRESS_THRESHOLD (8192ULL)
 #define MIN_COMPRESS_THRESHOLD (8ULL)
 
 /* This is the minimum journal file size */
--
2.13.3
