diff -up systemd-240/src/core/load-fragment.c.11~ systemd-240/src/core/load-fragment.c
--- systemd-240/src/core/load-fragment.c.11~	2018-12-22 14:52:34.550163871 +0100
+++ systemd-240/src/core/load-fragment.c	2018-12-22 14:53:51.664738838 +0100
@@ -4415,7 +4415,7 @@ static int open_follow(char **filename,
                 free_and_replace(*filename, target);
         }
 
-        f = fdopen(fd, "r");
+        f = fdopen(fd, "rm");
         if (!f) {
                 safe_close(fd);
                 return -errno;
