From ad97082cec9b5aee3f1224c172a6131d1a528389 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:38:31 +0100
Subject: [PATCH 05/39] journal: flush var/kmsg after starting.

---
 src/journal/journald.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff -Naur systemd-236/src/journal/journald.c systemd-236.tpg/src/journal/journald.c
--- systemd-236/src/journal/journald.c	2017-12-14 22:09:57.000000000 +0000
+++ systemd-236.tpg/src/journal/journald.c	2017-12-20 21:58:06.517597120 +0000
@@ -53,8 +53,6 @@
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server, true);
-        server_flush_dev_kmsg(&server);
 
         log_debug("systemd-journald running as pid "PID_FMT, getpid_cached());
         server_driver_message(&server, 0,
@@ -62,6 +60,9 @@
                               LOG_MESSAGE("Journal started"),
                               NULL);
 
+        server_flush_to_var(&server, true);
+        server_flush_dev_kmsg(&server);
+
         /* Make sure to send the usage message *after* flushing the
          * journal so entries from the runtime journals are ordered
          * before this message. See #4190 for some details. */
