diff -up systemd-240/units/systemd-timesyncd.service.in.23~ systemd-240/units/systemd-timesyncd.service.in
--- systemd-240/units/systemd-timesyncd.service.in.23~	2018-12-22 14:58:48.143282534 +0100
+++ systemd-240/units/systemd-timesyncd.service.in	2018-12-22 14:59:18.871900293 +0100
@@ -42,7 +42,7 @@ StateDirectory=systemd/timesync
 SystemCallArchitectures=native
 SystemCallErrorNumber=EPERM
 SystemCallFilter=@system-service @clock
-Type=notify
+Type=simple
 User=systemd-timesync
 WatchdogSec=3min
 
