diff -Naur systemd-235/meson.build systemd-235.tpg/meson.build
--- systemd-235/meson.build	2017-10-23 16:15:00.440716248 +0000
+++ systemd-235.tpg/meson.build	2017-10-06 08:18:04.000000000 +0000
@@ -2137,7 +2137,7 @@
 exe = executable('systemd-udevd',
                  systemd_udevd_sources,
                  include_directories : includes,
-                 c_args : ['-DLOG_REALM=LOG_REALM_UDEV', '-O3'],
+                 c_args : ['-DLOG_REALM=LOG_REALM_UDEV'],
                  link_with : [libudev_core,
                               libsystemd_network,
                               libudev_internal],
@@ -2154,7 +2154,6 @@
 exe = executable('udevadm',
                  udevadm_sources,
                  include_directories : includes,
-                 c_args : ['-O3'],
                  link_with : [libudev_core,
                               libsystemd_network,
                               libudev_internal],
diff -Naur systemd-235/src/udev/meson.build systemd-235.tpg/src/udev/meson.build
--- systemd-235/src/udev/meson.build	2017-10-06 08:18:04.000000000 +0000
+++ systemd-235.tpg/src/udev/meson.build	2017-10-23 16:16:35.123658915 +0000
@@ -131,7 +131,7 @@
         executable(prog[0].split('/')[0],
                    prog,
                    include_directories : includes,
-                   c_args : ['-DLOG_REALM=LOG_REALM_UDEV'],
+                   c_args : ['-DLOG_REALM=LOG_REALM_UDEV', '-O3'],
                    link_with : [libudev_internal],
                    install_rpath : udev_rpath,
                    install : true,
