diff -Naur systemd-232/src/udev/udevd.c systemd-232.tpg/src/udev/udevd.c
--- systemd-232/src/udev/udevd.c	2016-11-03 17:16:42.000000000 +0000
+++ systemd-232.tpg/src/udev/udevd.c	2016-11-04 11:52:30.669656097 +0000
@@ -1719,7 +1719,7 @@
         if (arg_daemonize) {
                 pid_t pid;
 
-                log_info("starting version " VERSION);
+                log_debug("starting version " VERSION);
 
                 /* connect /dev/null to stdin, stdout, stderr */
                 if (log_get_max_level() < LOG_DEBUG) {
