diff -up systemd-240/src/resolve/resolved.c.27~ systemd-240/src/resolve/resolved.c
--- systemd-240/src/resolve/resolved.c.27~	2018-12-22 15:03:12.241896673 +0100
+++ systemd-240/src/resolve/resolved.c	2018-12-22 15:04:45.420219661 +0100
@@ -45,6 +45,10 @@ static int run(int argc, char *argv[]) {
         /* Drop privileges, but only if we have been started as root. If we are not running as root we assume most
          * privileges are already dropped. */
         if (getuid() == 0) {
+                r = symlink("../run/systemd/resolve/resolv.conf", "/etc/resolv.conf");
+                if (r < 0 && errno != EEXIST)
+                        log_warning_errno(errno,
+                                          "Could not create /etc/resolv.conf symlink: %m");
 
                 /* Drop privileges, but keep three caps. Note that we drop those too, later on (see below) */
                 r = drop_privileges(uid, gid,
diff -up systemd-240/tmpfiles.d/etc.conf.m4.27~ systemd-240/tmpfiles.d/etc.conf.m4
--- systemd-240/tmpfiles.d/etc.conf.m4.27~	2018-12-21 19:53:33.000000000 +0100
+++ systemd-240/tmpfiles.d/etc.conf.m4	2018-12-22 15:03:12.241896673 +0100
@@ -13,9 +13,6 @@ L+ /etc/mtab - - - - ../proc/self/mounts
 m4_ifdef(`HAVE_SMACK_RUN_LABEL',
 t /etc/mtab - - - - security.SMACK64=_
 )m4_dnl
-m4_ifdef(`ENABLE_RESOLVE',
-L! /etc/resolv.conf - - - - ../run/systemd/resolve/stub-resolv.conf
-)m4_dnl
 C /etc/nsswitch.conf - - - -
 m4_ifdef(`HAVE_PAM',
 C /etc/pam.d - - - -
