diff -Naur systemd-232/units/systemd-timedated.service.in systemd-232.tpg/units/systemd-timedated.service.in
--- systemd-232/units/systemd-timedated.service.in	2016-11-03 17:16:42.000000000 +0000
+++ systemd-232.tpg/units/systemd-timedated.service.in	2016-11-04 11:50:42.888823575 +0000
@@ -11,6 +11,8 @@
 Documentation=http://www.freedesktop.org/wiki/Software/systemd/timedated
 
 [Service]
+Restart=always
+RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-timedated
 BusName=org.freedesktop.timedate1
 WatchdogSec=3min
