diff -Naur systemd-221/src/basic/macro.h systemd-221.tpg/src/basic/macro.h
--- systemd-221/src/basic/macro.h	2015-06-17 18:27:50.000000000 +0000
+++ systemd-221.tpg/src/basic/macro.h	2015-06-21 11:37:17.307312437 +0000
@@ -26,6 +26,10 @@
 #include <sys/types.h>
 #include <sys/uio.h>
 #include <inttypes.h>
+#ifdef __UCLIBC__
+/* uclibc does not implement mkostemp GNU extention */
+#define mkostemp(x,y) mkstemp(x)
+#endif
 
 #define _printf_(a,b) __attribute__ ((format (printf, a, b)))
 #define _alloc_(...) __attribute__ ((alloc_size(__VA_ARGS__)))
