diff -Naur systemd-238/src/udev/udev.conf systemd-238.tpg/src/udev/udev.conf
--- systemd-238/src/udev/udev.conf	2018-03-05 22:16:37.000000000 +0000
+++ systemd-238.tpg/src/udev/udev.conf	2018-03-10 15:58:59.774234921 +0000
@@ -3,4 +3,4 @@
 # udevd is also started in the initrd.  When this file is modified you might
 # also want to rebuild the initrd, so that it will include the modified configuration.
 
-#udev_log="info"
+udev_log="err"
