diff -Naur systemd-197/src/login/logind-dbus.c systemd-197.tpg/src/login/logind-dbus.c
--- systemd-197/src/login/logind-dbus.c	2012-12-23 23:29:39.964278288 +0000
+++ systemd-197.tpg/src/login/logind-dbus.c	2013-02-02 16:16:00.170545383 +0000
@@ -416,8 +416,8 @@
 
                 if (vtnr != 0)
                         return -EINVAL;
-
-        }
+        } else if (!isempty(tty) && s && seat_is_vtconsole(s))
+    		return -EINVAL;
 
         if (s) {
                 if (seat_can_multi_session(s)) {
