diff -Naur systemd-239/meson.build systemd-239.tpg/meson.build
--- systemd-239/meson.build	2018-10-01 17:54:16.989980000 +0000
+++ systemd-239.tpg/meson.build	2018-10-01 17:57:45.089978143 +0000
@@ -769,7 +769,7 @@
 
 enable_debug_hashmap = false
 enable_debug_mmap_cache = false
-foreach name : get_option('debug')
+foreach name : get_option('debug-extra')
         if name == 'hashmap'
                 enable_debug_hashmap = true
         elif name == 'mmap-cache'
diff -Naur systemd-239/meson_options.txt systemd-239.tpg/meson_options.txt
--- systemd-239/meson_options.txt	2018-06-22 11:11:49.000000000 +0000
+++ systemd-239.tpg/meson_options.txt	2018-10-01 17:57:02.008978738 +0000
@@ -46,7 +46,7 @@
        description : 'path to debug shell binary')
 option('debug-tty', type : 'string', value : '/dev/tty9',
        description : 'specify the tty device for debug shell')
-option('debug', type : 'array', choices : ['hashmap', 'mmap-cache'], value : [],
+option('debug-extra', type : 'array', choices : ['hashmap', 'mmap-cache'], value : [],
        description : 'enable extra debugging')
 option('memory-accounting-default', type : 'boolean',
        description : 'enable MemoryAccounting= by default')
diff -Naur systemd-239/sysusers.d/systemd.conf.m4 systemd-239.tpg/sysusers.d/systemd.conf.m4
--- systemd-239/sysusers.d/systemd.conf.m4	2018-06-22 11:11:49.000000000 +0000
+++ systemd-239.tpg/sysusers.d/systemd.conf.m4	2018-10-01 18:02:59.583973794 +0000
@@ -6,6 +6,15 @@
 #  (at your option) any later version.
 
 g systemd-journal   - -
+m4_ifdef(`ENABLE_NETWORKD',
+u systemd-network   - "systemd Network Management"
+)m4_dnl
+m4_ifdef(`ENABLE_RESOLVE',
+u systemd-resolve   - "systemd Resolver"
+)m4_dnl
+m4_ifdef(`ENABLE_TIMESYNCD',
+u systemd-timesync  - "systemd Time Synchronization"
+)m4_dnl
 m4_ifdef(`ENABLE_COREDUMP',
 u systemd-coredump  - "systemd Core Dumper"
 )m4_dnl
diff -Naur systemd-239/tmpfiles.d/systemd.conf.m4 systemd-239.tpg/tmpfiles.d/systemd.conf.m4
--- systemd-239/tmpfiles.d/systemd.conf.m4	2018-06-22 11:11:49.000000000 +0000
+++ systemd-239.tpg/tmpfiles.d/systemd.conf.m4	2018-10-01 18:04:20.970972668 +0000
@@ -17,9 +17,9 @@
 d /run/systemd/machines 0755 root root -
 d /run/systemd/shutdown 0755 root root -
 m4_ifdef(`ENABLE_NETWORKD',
-d /run/systemd/netif - - - -
-d /run/systemd/netif/links - - - -
-d /run/systemd/netif/leases - - - -
+d /run/systemd/netif 0755 systemd-network systemd-network -
+d /run/systemd/netif/links 0755 systemd-network systemd-network -
+d /run/systemd/netif/leases 0755 systemd-network systemd-network -
 )m4_dnl
 
 d /run/log 0755 root root -
