diff -Naur systemd-220/units/systemd-fsck-root.service.in systemd-220.tpg/units/systemd-fsck-root.service.in
--- systemd-220/units/systemd-fsck-root.service.in	2015-04-28 15:30:00.000000000 +0000
+++ systemd-220.tpg/units/systemd-fsck-root.service.in	2015-05-23 20:07:15.498282145 +0000
@@ -17,3 +17,5 @@
 RemainAfterExit=yes
 ExecStart=@rootlibexecdir@/systemd-fsck
 TimeoutSec=0
+StandardOutput=null
+StandardError=journal+console
diff -Naur systemd-220/units/systemd-fsck@.service.in systemd-220.tpg/units/systemd-fsck@.service.in
--- systemd-220/units/systemd-fsck@.service.in	2015-04-28 15:30:00.000000000 +0000
+++ systemd-220.tpg/units/systemd-fsck@.service.in	2015-05-23 20:07:39.552973030 +0000
@@ -18,3 +18,5 @@
 RemainAfterExit=yes
 ExecStart=@rootlibexecdir@/systemd-fsck %f
 TimeoutSec=0
+StandardOutput=null
+StandardError=journal+console
