diff -Naur systemd-219/units/systemd-timedated.service.in systemd-219.tpg/units/systemd-timedated.service.in
--- systemd-219/units/systemd-timedated.service.in	2015-02-11 17:28:06.000000000 +0000
+++ systemd-219.tpg/units/systemd-timedated.service.in	2015-02-25 21:38:22.257595004 +0000
@@ -11,6 +11,8 @@
 Documentation=http://www.freedesktop.org/wiki/Software/systemd/timedated
 
 [Service]
+Restart=always
+RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-timedated
 BusName=org.freedesktop.timedate1
 CapabilityBoundingSet=CAP_SYS_TIME
