--- systemd-196/src/shared/missing.h.0105~	2012-10-26 19:28:29.124141650 +0200
+++ systemd-196/src/shared/missing.h	2012-12-13 17:53:26.388457701 +0100
@@ -246,7 +246,8 @@ static inline int name_to_handle_at(int
 #  ifdef HAVE___SECURE_GETENV
 #    define secure_getenv __secure_getenv
 #  else
-#    error neither secure_getenv nor __secure_getenv are available
+#    warning neither secure_getenv nor __secure_getenv are available
+#    define secure_getenv getenv
 #  endif
 #endif
 
