diff -Naur systemd-208/units/systemd-vconsole-setup.service.in systemd-208.tpg/units/systemd-vconsole-setup.service.in
--- systemd-208/units/systemd-vconsole-setup.service.in	2013-08-13 20:02:52.000000000 +0000
+++ systemd-208.tpg/units/systemd-vconsole-setup.service.in	2014-04-06 08:57:16.602480401 +0000
@@ -10,8 +10,9 @@
 Documentation=man:systemd-vconsole-setup.service(8) man:vconsole.conf(5)
 DefaultDependencies=no
 Conflicts=shutdown.target
-After=systemd-readahead-collect.service systemd-readahead-replay.service
+After=systemd-readahead-collect.service systemd-readahead-replay.service systemd-udev-settle.service
 Before=sysinit.target shutdown.target
+Wants=systemd-udev-settle.service
 ConditionPathExists=/dev/tty0
 
 [Service]
