diff -Naur systemd-210/src/shared/macro.h systemd-210.tpg/src/shared/macro.h
--- systemd-210/src/shared/macro.h	2014-02-21 18:43:34.000000000 +0000
+++ systemd-210.tpg/src/shared/macro.h	2014-05-16 17:25:22.125837227 +0000
@@ -27,6 +27,11 @@
 #include <sys/uio.h>
 #include <inttypes.h>
 
+#ifdef __UCLIBC__
+/* uclibc does not implement mkostemp GNU extention */
+#define mkostemp(x,y) mkstemp(x)
+#endif
+
 #define _printf_(a,b) __attribute__ ((format (printf, a, b)))
 #define _alloc_(...) __attribute__ ((alloc_size(__VA_ARGS__)))
 #define _sentinel_ __attribute__ ((sentinel))
