diff -Naur systemd-210/src/shared/missing.h systemd-210.tpg/src/shared/missing.h
--- systemd-210/src/shared/missing.h	2014-02-21 02:07:23.000000000 +0000
+++ systemd-210.tpg/src/shared/missing.h	2014-05-16 17:23:18.170372873 +0000
@@ -303,7 +303,8 @@
 #  ifdef HAVE___SECURE_GETENV
 #    define secure_getenv __secure_getenv
 #  else
-#    error "neither secure_getenv nor __secure_getenv are available"
+#    warning neither secure_getenv nor __secure_getenv are available
+#    define secure_getenv getenv
 #  endif
 #endif
 
