diff -Naur systemd-238/src/basic/macro.h systemd-238.tpg/src/basic/macro.h
--- systemd-238/src/basic/macro.h	2018-04-11 16:07:49.531228000 +0000
+++ systemd-238.tpg/src/basic/macro.h	2018-04-11 16:09:45.745467487 +0000
@@ -48,10 +48,10 @@
 #define _weakref_(x) __attribute__((weakref(#x)))
 #define _alignas_(x) __attribute__((aligned(__alignof(x))))
 #define _cleanup_(x) __attribute__((cleanup(x)))
-#if __GNUC__ >= 7
+#if __GNUC__ >= 7 && __has_attribute(__fallthrough__)
 #define _fallthrough_ __attribute__((fallthrough))
 #else
-#define _fallthrough_
+#define _fallthrough_ ((void) 0)
 #endif
 /* Define C11 noreturn without <stdnoreturn.h> and even on older gcc
  * compiler versions */
