--- systemd-205/configure.ac.omv~	2013-07-05 23:56:37.470803488 +0200
+++ systemd-205/configure.ac	2013-07-05 23:56:54.585981423 +0200
@@ -44,7 +44,7 @@ AS_IF([test "x$host_cpu" = "xmips" || te
 LT_PREREQ(2.2)
 LT_INIT([disable-static])
 
-AS_IF([test "x$enable_static" = "xyes"], [AC_MSG_ERROR([--enable-static is not supported by systemd])])
+dnl AS_IF([test "x$enable_static" = "xyes"], [AC_MSG_ERROR([--enable-static is not supported by systemd])])
 AS_IF([test "x$enable_largefile" = "xno"], [AC_MSG_ERROR([--disable-largefile is not supported by systemd])])
 
 # i18n stuff for the PolicyKit policy files
