--- systemd-233/src/basic/log.c.omv~	2017-05-16 04:48:00.939986488 +0200
+++ systemd-233/src/basic/log.c	2017-05-16 04:50:04.870733177 +0200
@@ -701,6 +701,7 @@ int log_internal(
         return r;
 }
 
+DISABLE_WARNING_FORMAT_NONLITERAL
 int log_object_internalv(
                 int level,
                 int error,
@@ -856,6 +857,7 @@ int log_format_iovec(
         }
         return 0;
 }
+REENABLE_WARNING
 
 int log_struct_internal(
                 int level,
--- systemd-233/src/core/execute.c.omv~	2017-05-16 15:06:17.458736501 +0200
+++ systemd-233/src/core/execute.c	2017-05-16 15:07:07.032253251 +0200
@@ -2231,6 +2231,7 @@ static int acquire_home(const ExecContex
         return 1;
 }
 
+DISABLE_WARNING_FORMAT_NONLITERAL
 static int exec_child(
                 Unit *unit,
                 ExecCommand *command,
@@ -3018,6 +3019,7 @@ int exec_spawn(Unit *unit,
         *ret = pid;
         return 0;
 }
+REENABLE_WARNING
 
 void exec_context_init(ExecContext *c) {
         assert(c);
--- systemd-233/src/core/job.c.omv~	2017-05-16 15:04:31.546621636 +0200
+++ systemd-233/src/core/job.c	2017-05-16 15:04:59.591918411 +0200
@@ -745,6 +745,7 @@ static void job_print_status_message(Uni
         }
 }
 
+DISABLE_WARNING_FORMAT_NONLITERAL
 static void job_log_status_message(Unit *u, JobType t, JobResult result) {
         const char *format, *mid;
         char buf[LINE_MAX];
@@ -814,6 +815,7 @@ static void job_log_status_message(Unit
                    "RESULT=%s", job_result_to_string(result),
                    NULL);
 }
+REENABLE_WARNING
 
 static void job_emit_status_message(Unit *u, JobType t, JobResult result) {
 
--- systemd-233/src/core/service.c.omv~	2017-05-16 15:05:03.937964296 +0200
+++ systemd-233/src/core/service.c	2017-05-16 15:05:27.474212311 +0200
@@ -2636,6 +2636,7 @@ static void service_notify_cgroup_empty_
         }
 }
 
+DISABLE_WARNING_FORMAT_NONLITERAL
 static void service_sigchld_event(Unit *u, pid_t pid, int code, int status) {
         Service *s = SERVICE(u);
         ServiceResult f;
@@ -2937,6 +2938,7 @@ static void service_sigchld_event(Unit *
         if (cg_unified_controller(SYSTEMD_CGROUP_CONTROLLER) == 0 && set_isempty(u->pids))
                 service_notify_cgroup_empty_event(u);
 }
+REENABLE_WARNING
 
 static int service_dispatch_timer(sd_event_source *source, usec_t usec, void *userdata) {
         Service *s = SERVICE(userdata);
--- systemd-233/src/core/unit.c.omv~	2017-05-16 15:05:37.219314772 +0200
+++ systemd-233/src/core/unit.c	2017-05-16 15:05:48.306431185 +0200
@@ -1466,6 +1466,7 @@ static void unit_status_print_starting_s
         REENABLE_WARNING;
 }
 
+DISABLE_WARNING_FORMAT_NONLITERAL
 static void unit_status_log_starting_stopping_reloading(Unit *u, JobType t) {
         const char *format, *mid;
         char buf[LINE_MAX];
@@ -1502,6 +1503,7 @@ static void unit_status_log_starting_sto
                    LOG_MESSAGE("%s", buf),
                    NULL);
 }
+REENABLE_WARNING
 
 void unit_status_emit_starting_stopping_reloading(Unit *u, JobType t) {
         assert(u);
--- systemd-233/src/coredump/coredump.c.omv~	2017-05-16 15:48:31.972593715 +0200
+++ systemd-233/src/coredump/coredump.c	2017-05-16 15:48:52.428792990 +0200
@@ -710,6 +710,7 @@ static bool is_pid1_crash(const char *co
 
 #define SUBMIT_COREDUMP_FIELDS 4
 
+DISABLE_WARNING_FORMAT_NONLITERAL
 static int submit_coredump(
                 const char *context[_CONTEXT_MAX],
                 struct iovec *iovec,
@@ -837,6 +838,7 @@ log:
 
         return 0;
 }
+REENABLE_WARNING
 
 static void map_context_fields(const struct iovec *iovec, const char* context[]) {
 
--- systemd-233/src/locale/localed.c.omv~	2017-05-16 15:48:04.545326557 +0200
+++ systemd-233/src/locale/localed.c	2017-05-16 15:48:18.450461999 +0200
@@ -431,6 +431,7 @@ static int method_set_vc_keyboard(sd_bus
 
 #ifdef HAVE_XKBCOMMON
 
+DISABLE_WARNING_FORMAT_NONLITERAL
 _printf_(3, 0)
 static void log_xkb(struct xkb_context *ctx, enum xkb_log_level lvl, const char *format, va_list args) {
         const char *fmt;
@@ -438,6 +439,7 @@ static void log_xkb(struct xkb_context *
         fmt = strjoina("libxkbcommon: ", format);
         log_internalv(LOG_DEBUG, 0, __FILE__, __LINE__, __func__, fmt, args);
 }
+REENABLE_WARNING
 
 #define LOAD_SYMBOL(symbol, dl, name)                                   \
         ({                                                              \
