--- systemd-205/src/libsystemd-bus/test-bus-memfd.c.omv~	2013-07-05 23:24:32.220115973 +0200
+++ systemd-205/src/libsystemd-bus/test-bus-memfd.c	2013-07-05 23:25:01.058379221 +0200
@@ -145,6 +145,7 @@ int main(int argc, char *argv[]) {
         /* check content */
         assert_se(memcmp(buf, "ll", 2) == 0);
 
+#ifndef __UCLIBC__ /* uClibc doesn't have pwritev/preadv as of 0.9.33.2 */
         /* writev it out*/
         iov[0].iov_base = (char *)"ABC";
         iov[0].iov_len = 3;
@@ -167,6 +168,7 @@ int main(int argc, char *argv[]) {
         assert_se(memcmp(bufv[0], "ABC", 3) == 0);
         assert_se(memcmp(bufv[1], "DEF", 3) == 0);
         assert_se(memcmp(bufv[2], "GHI", 3) == 0);
+#endif
 
         sd_memfd_free(m);
 
