diff -Naur systemd-214/units/systemd-fsck-root.service.in systemd-214.tpg/units/systemd-fsck-root.service.in
--- systemd-214/units/systemd-fsck-root.service.in	2013-10-21 00:57:51.555743543 +0000
+++ systemd-214.tpg/units/systemd-fsck-root.service.in	2014-07-12 12:38:32.175942779 +0000
@@ -18,5 +18,6 @@
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=@rootlibexecdir@/systemd-fsck
-StandardOutput=journal+console
+StandardOutput=null
+StandardError=journal+console
 TimeoutSec=0
diff -Naur systemd-214/units/systemd-fsck@.service.in systemd-214.tpg/units/systemd-fsck@.service.in
--- systemd-214/units/systemd-fsck@.service.in	2013-10-21 00:57:51.555743543 +0000
+++ systemd-214.tpg/units/systemd-fsck@.service.in	2014-07-12 12:38:42.511220685 +0000
@@ -17,5 +17,6 @@
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=@rootlibexecdir@/systemd-fsck %f
-StandardOutput=journal+console
+StandardOutput=null
+StandardError=journal+console
 TimeoutSec=0
