diff -Naur systemd-234/units/systemd-timedated.service.in systemd-234.tpg/units/systemd-timedated.service.in
--- systemd-234/units/systemd-timedated.service.in	2017-07-12 14:01:10.023970656 +0000
+++ systemd-234.tpg/units/systemd-timedated.service.in	2017-07-27 22:10:21.235595763 +0000
@@ -11,6 +11,8 @@
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/timedated
 
 [Service]
+Restart=always
+RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-timedated
 BusName=org.freedesktop.timedate1
 WatchdogSec=3min
