diff -Naur systemd-208/src/libsystemd-bus/bus-control.c systemd-208.tpg/src/libsystemd-bus/bus-control.c
--- systemd-208/src/libsystemd-bus/bus-control.c	2013-08-13 20:02:46.000000000 +0000
+++ systemd-208.tpg/src/libsystemd-bus/bus-control.c	2014-06-29 22:03:47.696731104 +0000
@@ -25,7 +25,7 @@
 
 #include <stddef.h>
 #include <errno.h>
-
+#include "missing.h"
 #include "strv.h"
 
 #include "sd-bus.h"
diff -Naur systemd-208/src/libsystemd-bus/sd-bus.c systemd-208.tpg/src/libsystemd-bus/sd-bus.c
--- systemd-208/src/libsystemd-bus/sd-bus.c	2014-06-29 22:02:11.000000000 +0000
+++ systemd-208.tpg/src/libsystemd-bus/sd-bus.c	2014-06-29 22:04:09.040310842 +0000
@@ -33,7 +33,6 @@
 #include "macro.h"
 #include "strv.h"
 #include "set.h"
-#include "missing.h"
 
 #include "sd-bus.h"
 #include "bus-internal.h"
diff -Naur systemd-208/src/libsystemd-bus/test-bus-memfd.c systemd-208.tpg/src/libsystemd-bus/test-bus-memfd.c
--- systemd-208/src/libsystemd-bus/test-bus-memfd.c	2013-08-13 20:02:46.000000000 +0000
+++ systemd-208.tpg/src/libsystemd-bus/test-bus-memfd.c	2014-06-29 22:05:00.713714237 +0000
@@ -144,7 +144,8 @@
 
         /* check content */
         assert_se(memcmp(buf, "ll", 2) == 0);
-
+#ifndef __UCLIBC__
+       /* uClibc doesn't support pwritev and preadv, skip this check */
         /* writev it out*/
         iov[0].iov_base = (char *)"ABC";
         iov[0].iov_len = 3;
@@ -162,7 +163,7 @@
         iov[2].iov_base = bufv[2];
         iov[2].iov_len = 3;
         assert_se(preadv(fd, iov, 3, 0) == 9);
-
+#endif
         /* check content */
         assert_se(memcmp(bufv[0], "ABC", 3) == 0);
         assert_se(memcmp(bufv[1], "DEF", 3) == 0);
diff -Naur systemd-208/src/shared/missing.h systemd-208.tpg/src/shared/missing.h
--- systemd-208/src/shared/missing.h	2014-06-29 22:02:11.000000000 +0000
+++ systemd-208.tpg/src/shared/missing.h	2014-06-29 22:05:32.487577046 +0000
@@ -28,6 +28,10 @@
 #include <fcntl.h>
 #include <stdlib.h>
 #include <unistd.h>
+#ifdef __UCLIBC__
+/* uClibc has its own sysinfo */
+#define _LINUX_SYSINFO_H 1
+#endif
 #include <linux/oom.h>
 #include <linux/input.h>
 
diff -Naur systemd-208/src/shared/util.h systemd-208.tpg/src/shared/util.h
--- systemd-208/src/shared/util.h	2014-06-29 22:02:11.000000000 +0000
+++ systemd-208.tpg/src/shared/util.h	2014-06-29 22:05:50.814074679 +0000
@@ -41,6 +41,7 @@
 #include <unistd.h>
 #include <locale.h>
 #include <mntent.h>
+#include "missing.h"
 
 #include "macro.h"
 #include "time-util.h"
