diff -Naur systemd-230/src/core/macros.systemd.in systemd-230.tpg/src/core/macros.systemd.in
--- systemd-230/src/core/macros.systemd.in	2016-05-21 22:31:29.000000000 +0000
+++ systemd-230.tpg/src/core/macros.systemd.in	2016-06-07 16:59:51.803511156 +0000
@@ -21,6 +21,7 @@
 
 %_unitdir @systemunitdir@
 %_userunitdir @userunitdir@
+%_userpresetdir %{_prefix}/lib/systemd/user-preset
 %_presetdir @systempresetdir@
 %_udevhwdbdir @udevhwdbdir@
 %_udevrulesdir @udevrulesdir@
