diff -Naur systemd-230/src/journal/journald.conf systemd-230.tpg/src/journal/journald.conf
--- systemd-230/src/journal/journald.conf	2016-05-21 22:31:29.000000000 +0000
+++ systemd-230.tpg/src/journal/journald.conf	2016-05-24 17:11:18.853126631 +0000
@@ -13,17 +13,17 @@
 
 [Journal]
 #Storage=auto
-#Compress=yes
+Compress=yes
 #Seal=yes
 #SplitMode=uid
 #SyncIntervalSec=5m
 #RateLimitIntervalSec=30s
 #RateLimitBurst=1000
-#SystemMaxUse=
+SystemMaxUse=150M
 #SystemKeepFree=
 #SystemMaxFileSize=
 #SystemMaxFiles=100
-#RuntimeMaxUse=
+RuntimeMaxUse=150M
 #RuntimeKeepFree=
 #RuntimeMaxFileSize=
 #RuntimeMaxFiles=100
